package.json

```diff
@@ -2,23 +2,30 @@
   "name": "browserify-rsa",
   "version": "4.0.1",
   "description": "RSA for browserify",
+  "bugs": {
+    "url": "https://github.com/crypto-browserify/browserify-rsa/issues"
+  },
+  "license": "MIT",
+  "files": [
+    "index.js"
+  ],
   "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com:crypto-browserify/browserify-rsa.git"
+  },
   "scripts": {
-    "test": "node test.js | tspec"
+    "lint": "standard",
+    "test": "npm run lint && npm run unit",
```

Comment:
but what if another person prefer another output that based on TAP output?
I'm not against tap-spec, I just think that output should be raw, in this case anybody can decide what output he want to get.